routes/upload-files.ts

```diff
@@ -25,41 +40,124 @@ interface FileUploaderRequest extends Request {
   }];
 }
 
-function uploadPoster(req: FileUploaderRequest, res: any) {
+async function uploadAvatar(req: Request, res: Response): Promise<void> {
+  const file = req.body.fileBody;
+  const headers = req.headers;
+  const filename = headers.filename || new Date().getTime().toString();
+  const pathName = `uploads/avatars/${filename}`;
+  const allowedMimeTypes = ['image/gif', 'image/jpeg', 'image/pjpeg', 'image/x-png', 'image/png', 'image/svg+xml'];
+
+  if (allowedMimeTypes.indexOf(headers['content-type']) === -1) {
+    const status = 500;
+
+    res.status(status).send({message: 'Unsupported type of file'});
+  }
+
+  const savedFile = await saveFile(`./${pathName}`, file);
+
+  if (!savedFile.isSuccess) {
+    const status = 500;
```

Comment:
use HTTPStatus

Reply:
I'll do it for all project  in a separate PR